swmmio/core.py

```diff
@@ -498,6 +499,20 @@ def headers(self):
 
         return self._rpt_section_details
 
+    @property
+    def external_outflow_volume(self):
```

Comment:
This is really cool! It is making think, though, that we need to provide coverage for the whole `Flow Routing Continuity` section of the rpt file. What do you think? 

If you agree, maybe we can tackle that in a separate issue. 
Then we could extend that logic to provide coverage for the rest of the rpt sections that follow this structure.

Reply:
This could be handled in #194